docs/en_us/platform_api/source/shared/courses_response_values.rst

```diff
@@ -0,0 +1,223 @@
+.. _Course:
+
+Course
+*******
+
+* count (integer): The number of courses in the catalog.
+* next (string): The URL for the next page of results.
+* previous (string): The URL for the previous page of results.
+* results (array): See :ref:`Results`.
+
+.. _Results:
+
+Results
+*******
+
+Parent: ``course``
```

Comment:
So this is what I see at https://stage-edx-discovery.edx.org/api/v1/catalogs/1/courses/:

```
{
    "count": 975,
    "next": "https://stage-edx-discovery.edx.org/api/v1/catalogs/1/courses/?limit=20&offset=20",
    "previous": null,
    "results": [
        {
            "key": "zxcvzxczxcv+zxcvzxcv",
            "title": "aqdaszzxcvzxcvzxcv",
            "short_description": null,
            "full_description": null,
            "level_type": null,
            "subjects": [],
            "prerequisites": [],
            "expected_learning_items": [],
            "image": null,
            "video": null,
```

I read this as there's an API called Courses, and it has the children "count", "next", "previous", and "results". Then "results" has lots of children, including "key", "title", etc. Can you help me understand this better?